prow/kube/prowjob.go

```diff
@@ -79,6 +79,8 @@ type ProwJobStatus struct {
 	URL             string       `json:"url,omitempty"`
 	PodName         string       `json:"pod_name,omitempty"`
 	BuildID         string       `json:"build_id,omitempty"`
+	UUID            string       `json:"uuid,omitempty"`
```

Comment:
I am not arguing specifically for Jenkins but merely for the PJ API. There is already pj.Metadata.Name and pj.Metadata.UID - both are unique. Adding a third field doesn't seem right.